scripts/scil_smooth_surface.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+Script to smooth surface from a Laplacian blur.
```

Comment:
This is a complex script, is there any corner case, recommendation, to do and not to do coming with it?
Like right now the default are made for a human cortical surface I assume ? It would be important to mention it.

You could also mention range of value if someone needs to fine-tune it a bit, for example I would have though that the parameters step_size would have been between 0.1 and 1 and nb_steps would have been around 20-50.
More details would be important for future user